cmake/toolchain/xtools/target.cmake

```diff
@@ -2,18 +2,22 @@
 
 set(CROSS_COMPILE_TARGET_arm         arm-zephyr-eabi)
 set(CROSS_COMPILE_TARGET_nios2     nios2-zephyr-elf)
-set(CROSS_COMPILE_TARGET_riscv   riscv32-zephyr-elf)
+set(CROSS_COMPILE_TARGET_riscv   riscv64-zephyr-elf)
```

Comment:
do we support both 32-bit and 64-bit variants of riscv? @nategraff-sifive

Reply:
This is copied straight from `cmake/toolchain/zephyr/0.10/target.cmake`.

Reply:
We support both now actually. I'm pretty sure the way this works now is the 64-bit toolchain also supports 32-bit. Paging @galak, maybe you know if this part of the cmake scripts needs changing?

Reply:
yes, the riscv64 target supports both 32-bit and 64-bit.